dev/io.openliberty.java.internal_fat/fat/src/io/openliberty/java/internal/fat/Java17CDITest.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
```

Comment:
Please update the copyright date since this is a new file.

Reply:
Done